packages/react-data-grid/src/Canvas.tsx

```diff
@@ -261,6 +293,16 @@ export default function Canvas<R>({
     return <Row<R> {...rendererProps} />;
   }
 
+  function getRowRef(row: R, idx: number) {
```

Comment:
Summary rows `ref` will never reset from what I can tell. Consider a case
1) 10 rows + 2 summary rows are renderered
2) rowCount is changed to 30. This will override summary row refs

I am quite sure this will happen but we can double check. We may have to use 2 maps 

> React will call the ref callback with the DOM element when the component mounts, and call it with null when it unmounts.

Reply:
you are right, I will use another way for the summary rows ref.